mlinspect/monkeypatching/README.md

```diff
@@ -54,6 +54,7 @@ compare(networkx.to_dict_of_dicts(extracted_dag), networkx.to_dict_of_dicts(expe
 | `('pandas.core.frame', '__getitem__')`, arg type: strings | Projection|
 | `('pandas.core.frame', '__getitem__')`, arg type: series | Selection |
 | `('pandas.core.frame', 'dropna')` | Selection      |
+| `('pandas.core.frame', 'sample')` | Selection      |
```

Comment:
I think we shouldn't use the selection operator for this but introduce a new one that captures its semantics better. Something along the lines of `OperatorTypes.RESAMPLE`, what do you think? 

The existing inspections and checks need a minor update then also to handle this new operator type. And then we should add new tests for the inspections and checks where it makes sense to check that the new behavior works. That's mainly `NoBiasIntroducedFor` and `HistogramForColumns`. The tests for that are in `test/inspections/test_histogram_for_columns.py` and `test/check/test_no_bias_introduced_for.py`.

Would you be willing to add that also?

Reply:
I just noticed that the `frac` option is allowed to be bigger than 1. Than it obviously isn't a selection anymore. So yes, adding a new OperatorType seems to be a good idea. But not for this evening.